### DISCUSSION TITLE
In game reset

### DISCUSSION THREAD

**Discussion by NoodleTsunami** [Polprzewodnikowy/N64FlashcartMenu#92] [Q&A] [answered]
Is there a way to perform an In Game Reset with the controller to go back to the Menu?  Kind of like a button combo uncommonly used.  Thanks!

**Comment by Polprzewodnikowy** [comment, marked as answer]
No, N64 hardware architecture doesn't really allow to have an easy way to do that. Every game would need to have a separate dedicated patch to hook into controller routine. It's too much work and new homebrew games or romhacks are guaranteed to be unsupported.

Other method for in-game reset is to install RGB or HDMI mod in your console. Those usually hooks to the first controller port and can listen on button combo. It's far better solution because it doesn't rely on modifying game code.
